client/blocks/reader-combined-card/style.scss

```diff
@@ -82,6 +82,23 @@
 	display: flex;
 	flex-direction: row;
 	margin-bottom: 20px;
+	position: relative;
+
+	&.is-selected {
+		&::before {
+			content: '';
+			position: absolute;
```

Comment:
eh? these are pulled directly from the existing styles. https://github.com/Automattic/wp-calypso/blob/master/client/blocks/reader-post-card/style.scss#L37